### ISSUE TITLE
Deferral Schedule (Date isssue)

### ISSUE THREAD

**Issue by niranjand22** [3ktech/General#441]
Hi Niranjan,

Currently there are two reports in BC that refers Prepayement amount in Chart of Accounts,
1.	Purchasing Deferral Summary
2.	G/L Deferral Summary.
Date filter available for these reports is "as on date". We have taken report as on 31st March,2023, but april entries are appeared in the reports.

Kindly check the same and update the report where "as on 31st March.2023" report shall give data with actual pending prepayments as on 31st march end with exact match with TB balance
(i.e. entries after mar should not appear in the reports)

**Comment by niranjand22** [comment]
Client will connect today

**Comment by niranjand22** [comment]
the client was busy last week.they will connect today or  tomorrow

**Comment by niranjand22** [comment]
after providing date filter till march..still april entries are coming in report

![Image](https://github.com/3ktech/General/assets/116243958/b8d9ee68-e76b-4419-b4ce-6e4f5bc7b903)

**Comment by niranjand22** [comment]
[document (43).pdf](https://github.com/3ktech/General/files/11595595/document.43.pdf)

**Comment by niranjand22** [comment]
![Image](https://github.com/3ktech/General/assets/116243958/29977a0d-0f96-4ed8-80c8-326eddc0f994)

**Comment by niranjand22** [comment]
I checked ..this is standard reort and there is no any custom field

**Comment by niranjand22** [comment]
@panduc  Lets connect tomorrow on this case

**Comment by niranjand22** [comment]
We will connect with pandu today

**Comment by niranjand22** [comment]
Mr Pandu will anyalse and give the hours

**Comment by panduc** [comment]
Hi @niranjand22  after analyzing the standard the design is based date filter only after the specified if any entries is showing in remaining amount deferred.

Scenario Purchasing Deferral Summary Report(1702)

Case1:


![Image](https://github.com/3ktech/General/assets/110709312/5c206e4f-0d79-4bc2-845c-7f06605a16ec)


Case2: 


![Image](https://github.com/3ktech/General/assets/110709312/ebcd52c6-8f7a-4bfd-ac9c-ba4a6f19e74a)

**Comment by panduc** [comment]
@niranjand22 as discussed please finalize expected output based on that i will provide estimation.

**Comment by niranjand22** [comment]
@panduc  I connected with Ms Swathi..they want to filter out any entry other that AS of Date.

eg If as of date is 31st the don't want any entry to be printed for april

**Comment by panduc** [comment]
@niranjand22 as discussed for both reports modification requires 3 hours.

Report 1702 and Report 1700

**Comment by niranjand22** [comment]
@panduc  you can work on this

**Comment by panduc** [comment]
Developed the changes need the environment to deploy and test. Let's connect tomorrow.

**Comment by niranjand22** [comment]
tested in custom report layout, works fine will showcase to client and update

**Comment by niranjand22** [comment]
they want same clumns as standard report @panduc

**Comment by panduc** [comment]
@niranjand22 the changes has been done. Please check and confirm

**Comment by niranjand22** [comment]
Technical is not avaialble for further changess

**Comment by niranjand22** [comment]
Hi @panduc , Can you please make the remaining changes so we can close this

**Comment by panduc** [comment]
@niranjand22 as mentioned to you. I've shared it to Sandhya. Please get the further update from her.

**Comment by audhi3k** [comment]
Mr @panduc and Mr Niranjan can you update the status

**Comment by panduc** [comment]
@niranjand22 as discussed the changes are enabled in Sandbox_india. Please test and confirm.


1. Purchase Deferral Summary Cust - Id 50111
2. G/L Deferral Summary Cust - ID 50110

Please test it in LETRIM INTELLIGENCE SERVICES PRIVATE LIMITED Company.

CC: @audhi3k

**Comment by panduc** [comment]
@niranjand22  as you mentioned deployed into production. Please check and confirm.

CC: @audhi3k

**Comment by niranjand22** [comment]
I have tested, It is working as expected. have asked client to check and confrim

**Comment by niranjand22** [comment]
@panduc  Please add grand total in the last for Amount Recoshgnised, remaining amount fedred and total amount defred.

**Comment by panduc** [comment]
@niranjand22  Grand total has been added to both the reports and deployed Production. Please check and Confirm.

**Comment by niranjand22** [comment]
Balance is not matching in report..will connect with sruthi for this..tomoroww

**Comment by niranjand22** [comment]
Three entries were missing from report because client has forget to add defferal table while doing the transaction.

Explained to client. He is okay with report. And confirmed for closure.

cc @audhi3k
